.requirements

```diff
@@ -8,5 +8,5 @@ RESTY_OPENSSL_VERSION=1.1.1o
 RESTY_PCRE_VERSION=8.45
 RESTY_LMDB_VERSION=master
 LIBYAML_VERSION=0.2.5
-KONG_BUILD_TOOLS_VERSION=4.28.0
+KONG_BUILD_TOOLS_VERSION=master
```

Comment:
Master branche are mutable. Can we wait for a new version of KBT and use the latest version.

Reply:
Mentioned in this comment here -> https://github.com/Kong/kong/pull/8842#discussion_r883556550

This is only a temporary change and will be reverted once KBT released a version with https://github.com/Kong/kong-build-tools/pull/475 included.